pyuvdata/utils.py

```diff
@@ -816,6 +837,7 @@ def get_antenna_redundancies(antenna_numbers, antenna_positions, tol=1.0, includ
         antenna_positions:  Antenna position vectors in meters. Cartesian frame  (float), shape (Nants, 3)
         tol = (float)   Redundancy tolerance in meters. (float)
         include_autos =  Include autocorrelations (bool). Default is false
+        with_conjugates = Include baselines that are redundant if flipped. Default is false.
```

Comment:
This is confusing because it's unclear what the conjugated or unconjugated means for antennas. I'd probably lean toward always getting all possible pairs, but at a minimum we need to document what direction is assumed (i.e. is it the vector from the smaller to larger antenna number or the other way around?).

Reply:
The antenna method just makes baseline vectors from antenna positions and numbers and then uses `get_baseline_redundancies`. It defines baselines such that the larger antenna number is first, instead of making a condition on the baseline orientation, which is why it has the same result as `get_baseline_redundancies` when run without the `with_conjugate` option. I guess that behavior is a bit unclear... it might be more useful to have it return antenna pair tuples equivalent to the `with_conjugates` mode.